web/assets/js/actions.ts

```diff
@@ -358,6 +369,41 @@ class Actions {
     );
   }
 
+  /**
+   * Claims that the game timed out.
+   * Errors out if the claim timeout request is invalid,
+   * and returns the game object if the request is valid (game ended)
+   * @param gameID
+   */
+  async claimTimeout(gameID: string): Promise<Game> {
+    // Make the request
+    const timeoutResponse: BroadcastTxCommitResult =
+      (await this.wallet?.callMethod(
+        chessRealm,
+        'ClaimTimeout',
+        [gameID],
+        TransactionEndpoint.BROADCAST_TX_COMMIT,
+        undefined,
+        {
+          gasFee: defaultTxFee,
+          gasWanted: defaultGasWanted
+        }
+      )) as BroadcastTxCommitResult;
```

Comment:
```suggestion
    const timeoutResponse = await this.callMethod(
        chessRealm,
        'ClaimTimeout',
        [gameID]
      );
```

(copy and paste, don't commit directly, as I am sure prettier doesn't like how I formatted it. may need `git pull --no-rebase origin master`)

Reply:
@zivkovicmilos could you check this action file? I think there is a lot of conflicts and I wouldn't like to destroy your and Alex's work.

Reply:
[85305a6](https://github.com/gnolang/gnochess/pull/54/commits/85305a61323a7867da098a9b24ab286bb6aeb24e)